app.js

```diff
@@ -29,7 +29,12 @@ app.use((err, req, res, next) => {
     res.status(err.status).send({ msg: err.msg });
   }
   if (err.code === "23503") {
-    res.status(404).send({ msg: "username not found" });
+    if (err.detail === 'Key (author)=(kevin) is not present in table "users".')
```

Comment:
These should be more generic, firstly because this ends up needing lots of different scenarios, really specifically here for every review_id or author. It also exposes table names in the message which could lead to SQL vulnerabilities.

Bad request and not found error messages are sufficient.